### ISSUE TITLE
Browser UI focus ring should be Brave color cross-platform

### ISSUE THREAD

**Issue by petemill** [brave/brave-browser#1190]
Focus rings in Chromium are determined by the Native theme, which can differ per-platform. Right now this seems to be the system setting for light theme, and hard-coded for dark / private theme.
We should change this to hard-coded for each / all themes since our spec specifies specific colors.

Separating this since we don't yet have any overriden Native Themes (dark or light, windows or linux or mac).

**Comment by btlechowski** [comment]
Verification passes on

Brave | 0.55.10 Chromium: 70.0.3538.22 (Official Build) beta (64-bit)
-- | --
Revision | ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364-refs/branch-heads/3538@{#468}
OS | Windows 7

Checked URL Bar for themes Light and Dark. 
Checked Windows: Normal. Private, Private with Tor and Guest
